translate/src/modules/entitydetails/components/Metadata.tsx

```diff
@@ -233,6 +263,7 @@ export function Metadata({
       <GroupComment comment={entity.group_comment} />
       <ResourceComment comment={entity.resource_comment} key={entity.pk} />
       <FluentAttribute entity={entity} />
+      <EntityCreatedDate dateCreated={entity.date_created} />
```

Comment:
Please move this element to the very bottom, under `EntityContext`, because it's the least relevant.